### PR TITLE
Feature/152798589/paystack integration

### PR DESCRIPTION
**Pull request by williamolojede**
## What does this PR do?
- Add paystack and wallet as an alternative payment option

## Description of Task to be completed?
User can checkout paystack or their account wallet - which also allows he/she to transfer funds to other users.

## How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
[#152798589](https://www.pivotaltracker.com/story/show/152798589)

## Screenshots (if appropriate)
image

checkout

#### Questions: 
N/A

### CODE REVIEW

File: .meteor/versions [hunk 1]
```diff
@@ -105,7 +105,6 @@ mdg:validated-method@1.1.0
 mdg:validation-error@0.5.1
 meteor@1.7.2
 meteor-base@1.1.0
-meteor-tool@1.5.4
```
Comment (ooduye) [inline .meteor/versions:108]: Why did you delete meteor tool

File: imports/plugins/custom/payments-paystack/client/checkout/paystack.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<template name="paystackPaymentForm">
+  {{#autoForm schema=PaystackPayment id="paystack-payment-form" doc=showdoc}}
+  <!-- <div class="row">
```
Comment (ooduye) [inline imports/plugins/custom/payments-paystack/client/checkout/paystack.html]: Commented code should be gotten rid of completely

File: imports/plugins/custom/payments-paystack/client/checkout/paystack.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+const submitting = false;
+
+function uiEnd(template, buttonText) {
+  template.$(":input").removeAttr("disabled");
+  template.$("#btn-complete-order").text(buttonText);
+  return template.$("#btn-processing").addClass("hidden");
+}
+
+function paymentAlert(template, errorMessage) {
+  // return $(".alert").removeClass("hidden").text(errorMessage);
```
Comment (ooduye) [inline imports/plugins/custom/payments-paystack/client/checkout/paystack.js]: Why this comment too?! And it almost seems like nothing would be returned

File: imports/plugins/custom/wallet/server/init.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Hooks.Events.add("afterCoreInit", () => {
+  Reaction.addRolesToGroups({
+    allShops: true,
+    groups: ["owner"],
+    roles: ["account/verify"]
+  });
+});
+
+
+// import { Meteor } from "meteor/meteor";
```
Comment (ooduye) [inline imports/plugins/custom/wallet/server/init.js]: commented out code

File: imports/plugins/custom/wallet/server/publications/walletPublication.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { Meteor } from "meteor/meteor";
+import { Accounts, Wallets, WalletHistories  } from "/lib/collections";
```
[the remark this answers is not in the document]
Comment (ooduye) [reply imports/plugins/custom/wallet/server/publications/walletPublication.js]: multiple spaces before }

File: imports/plugins/custom/payments-paystack/client/checkout/paystack.js (hunk 3 above)
Comment (williamolojede) [reply imports/plugins/custom/payments-paystack/client/checkout/paystack.js]: will remove the comment, it doesn't have to return aything to work tho

File: imports/plugins/custom/wallet/server/init.js (hunk 4 above)
Comment (williamolojede) [reply imports/plugins/custom/wallet/server/init.js]: will fix
